README.md

```diff
@@ -28,3 +28,23 @@ CHROMEDRIVER_PATH=/path/to/chromedriver bundle exec rake
 
 If you do not have chromedriver installed, first execute
 `bin/install-chromedriver`, which will install it for you and report the path.
+
+## Webpacker
+
+We do not use the Rails asset pipeline: we use
```

Comment:
`,` instead of `:` ?

Reply:
Should we specify why we decided to use webpack/webpacker?

Reply:
Yes — I think this can be a good ADR https://trello.com/c/oivKkA7C/667-establish-architectural-decision-records-adrs-for-apply

Reply:
> `,` instead of `:` ?

Guilty as charged 🙂. This is a bad colon. Will amend